buildman/manager/ephemeral.py

```diff
@@ -922,11 +922,11 @@ def _work_checker(self):
                 logger.debug("Build job %s scheduled.", job_id)
             else:
                 logger.warning(
-                    "Unsuccessful schedule. Build ID: %s. Retry restored.",
+                    "Unsuccessful schedule. Build ID: %s. Check build executor service.",
```

Comment:
@kleesc  looks good. If we get a lot of warnings because of this we might have to think about adding a retry counter here